docs/pages/setup/admin/daemon.mdx

```diff
@@ -33,89 +31,102 @@ $ readlink /sbin/init
-● teleport.service - Teleport SSH Service
-     Loaded: loaded (/lib/systemd/system/teleport.service; enabled; vendor preset: enabled)
-     Active: active (running) since Mon 2022-04-18 18:33:41 UTC; 41s ago
-   Main PID: 442 (teleport)
-      Tasks: 9 (limit: 1116)
-     Memory: 116.9M
-     CGroup: /system.slice/teleport.service
-             └─442 /usr/local/bin/teleport start --pid-file=/run/teleport.pid
-
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] SSH proxy service 9.0.4:v9.0.4-0-gf577413>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] SSH proxy service 9.0.4:v9.0.4-0-gf577413>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROC:1]    The new service has started successfully.>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:AGE] Starting reverse tunnel agent pool. servi>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:PRO] Starting Kube proxy on . service/service.>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [DB:SERVIC] Starting Postgres proxy server on 0.0.0.0>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [DB:SERVIC] Starting Database TLS proxy server on 0.0>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] Starting proxy gRPC server on [::]:3080. >
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] Starting TLS ALPN SNI proxy server on [::>
-Apr 18 18:33:51 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:51Z WARN [PROXY:1]   Restart watch on error: empty proxy list.>
-```
+  ```code
+  $ sudo systemctl status teleport | grep Active
+  Active: active (running) since Mon 2022-04-18 18:33:41 UTC; 41s ago
+  ```
+  </TabItem>
+
+  <TabItem label="Source Installation">
+  In order to more readily support systemd configurations, Teleport is capable of recommending and assisting with the installation of a unit file.
+
+  Enter the following `teleport` command to have Teleport output a systemd unit file to `stdout` to be reviewed and applied as needed:
+
+  ```code
+  $ teleport install systemd
+  ```
+
+  If the output describes a good configuration for your environment, enter the command once again using the `-o` flag to
+  write the output to a file within one of systemd's accepted unit file load paths. For most use cases, we recommend using the path `/etc/systemd/system/teleport.service` for your unit file:
+
+  ```code
+  $ sudo teleport install systemd -o /etc/systemd/system/teleport.service
+  ```
 
-The next time you restart your host, `systemd` will run the `teleport` daemon
-automatically.
+  Enable the `teleport` service daemon using `systemctl` so that systemd can place it in its dependancy tree. This also ensures that the `teleport` service will be automatically restarted on any potential reboot:
```

Comment:
```suggestion
  Enable the `teleport` service daemon using `systemctl` so that systemd can place it in its dependency tree. This also ensures that the `teleport` service will be automatically restarted on any potential reboot:
```